packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js

```diff
@@ -1478,6 +1478,171 @@ export function createViewTransitionInstance(
   };
 }
 
+type EventListenerOptionsOrUseCapture =
+  | boolean
+  | {
+      capture?: boolean,
+      once?: boolean,
+      passive?: boolean,
+      signal?: AbortSignal,
+      ...
+    };
+
+type StoredEventListener = {
+  type: string,
+  listener: EventListener,
+  optionsOrUseCapture: void | EventListenerOptionsOrUseCapture,
+};
+
+export type FragmentInstance = {
+  _children: Set<Element>,
+  _eventListeners: Array<StoredEventListener>,
+  parentInstance: Instance | Container,
+  appendChild(child: Element): void,
+  removeChild(child: Element): void,
```

Comment:
`appendChild`/`removeChild` doesn't make much sense as publicly accessible APIs. A fragment doesn't even have a single parent and the children could be spread out.

It's also not something you should mess with in user space since it's an internal set which fragile semantics. E.g. this implementation doesn't preserve any kind of ordering guarantees.

There's no need for the internal methods to operate on these methods so they don't need to exist. Just have `appendChildToFragmentInstance` add to the `_children` Set directly.